packages/@uppy/dashboard/src/components/FileItem/FileProgress/index.js

```diff
@@ -42,44 +37,150 @@ function progressIndicatorTitle (props) {
   return ''
 }
 
+// function PauseResumeCancelIcon (props) {
```

Comment:
I'd vote that, when we're ready to commit to master, it's time to remove large old functions as comments like this one and start relying on Git for history to retrieve them

Reply:
Yes, I will do that before merging, I’m keeping this wile working on the PR to make sure I’ve moved all the parts and didn’t forget something.